test/prog/dftb+/dispersion/2H2O_dftd4/dftb_in.hsd

```diff
@@ -40,7 +40,14 @@ Hamiltonian = DFTB {
     s9 = 1.0
     s8 = 0.61
     CutoffInter = 60
-    CutoffCount = 34
+    CoordinationNumber = cov {
+       Cutoff = 34
+    }
+    ChargeModel = EEQ {
+       CoordinationNumber = erf {
```

Comment:
We should probably capitlize the coordination number containers as well (e.g. `Erf`, `Cov`, ...) Similarly also in the manual then.